### ISSUE TITLE
Suggestion: WWHD tweaks to Triforce Quest, first-person movement & Magic Armor

### ISSUE THREAD

**Issue by AgitationSkeleton** [WideBoner/betterww#12]
Wind Waker HD simplifies the triforce quest by only requiring three of the shards to be obtained via chart translation. (Which ones specifically I'm not entirely sure, it's been a while since I played WWHD)

It also unlocks player movement while in first-person (C-up, using a bow, hookshot or grappling hook). 

Magic Armor is buffed by using rupees per hit instead of magic, and can be used while sailing.

Apologies for writing this here, not sure where else to make suggestions. Hopefully these are possible within the scope of romhacking. Thank you!

**Comment by chrisasma103** [comment]
Yeah, this seems pretty ambitious, but I would love to see this happen!

**Comment by WideBoner** [comment]
I am keeping this open as a to-do list of things that I want to add. Triforce quest is at the top of the list, and the other suggestions are very much worth adding. Thanks

**Comment by Xiarno** [comment]
Will this ever be added? Especially the first-person movement?
